src/sentry/tasks/relay.py

```diff
@@ -124,7 +138,7 @@ def schedule_update_config_cache(
             "One of organization_id, project_id, public_key has to be provided, not many."
         )
 
-    if projectconfig_debounce_cache.check_is_debounced(public_key, project_id, organization_id):
+    if projectconfig_debounce_cache.is_debounced(public_key, project_id, organization_id):
```

Comment:
What if we move this as a function into the debouncecache, with a lambda that schedules? Basically restoring `check_is_debounced`. That would allow us to remove all methods introduced in #34548 from the public interface.

Reply:
I see. If that's the case, I don't like the approach. `check_is_debounced` should only care about its class' functionality, and don't care about scheduling functionality since that's the caller's problem.